contrib/intel/jenkins/Jenkinsfile

```diff
@@ -31,16 +31,21 @@ def slurm_batch(partition, node_num, output, command) {
 }
 
 def run_fabtests(stage_name, partition, node_num, prov, util=null,
-                 user_env=null) {
+                 user_env=null, way=null) {
   def command = "python3.9 ${RUN_LOCATION}/runtests.py"
   def opts = "--prov=${prov} --test=fabtests"
+  def modes = BUILD_MODES
   if (util)
     opts = "${opts} --util=${util}"
 
   if (user_env)
     opts = "${opts} --user_env ${user_env}"
 
-  for (mode in BUILD_MODES) {
+  if (way)
```

Comment:
this needs curly brackets around it. Right now youre forcing fabtests to only do reg since the modes = ["reg"] line is not protected in this if.